plugins/inputs/exec/sample.conf

```diff
@@ -28,3 +28,7 @@
   ## them here:
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
   # data_format = "json"
+
+  ## Run the exec plugin in debug mode
+  ## If set to true, exec will not truncate the command's stderr output.
+  # debug = false
```

Comment:
Could you also add this to the sample config in the readme?

Reply:
Done! Did that by running `make docs`